src/IssueViz/Monikers.imagemanifest

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- This file was generated by the ManifestFromResources tool.-->
```

Comment:
FYI Mads wrote an extension for VS 2019 for this, so that's what I used :) 

https://marketplace.visualstudio.com/items?itemName=MadsKristensen.ImageManifestTools